code/game/objects/items/devices/camera_bug.dm

```diff
@@ -20,41 +21,44 @@
 		/obj/item/weapon/storage
 	)
 
+/obj/item/device/camera_bug/do_attack_animation(atom/target, mob/user)
```

Comment:
This doesn't do anything. `do_attack_animation()` is called on the attacker, not the tool.

Reply:
I got rid of this bit, and the hitsound line since they both don't do anything in this situation.